debezium-connector-mysql/src/test/resources/ddl/enum_column_test.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE `test_stations_10` (
 INSERT INTO test_stations_10 (`name`, `type`) values ( 'ha Tinh 7', 'station' );
 
 ALTER TABLE `test_stations_10`
-    MODIFY COLUMN `type` ENUM('station', 'post_office', 'plane', 'ahihi_dongok', 'now')
+    MODIFY COLUMN `type` ENUM('station', 'post_office', 'plane', 'ahihi_dongok', 'now', 'test', 'a,b')
```

Comment:
Ok, so as we're going the extra mile already, it'd also be nice to have a test for an enum constant with an (escaped) tick: `'a,\'b` (assuming backslashes are used for escaping).